pkg/repo/error.go

```diff
@@ -0,0 +1,13 @@
+package repo
+
+import (
+	"errors"
+
+	"golang.org/x/text/language"
+)
+
+var ErrNotFound = errors.New("entity not found")
+
+type LoadMessagesOpts struct {
```

Comment:
Why did you move LoadMessagesOpts?

Reply:
Ops..
For some reason I had this file already in my local machine. mb I have created it when i was working on different ticket.
I will remove this error.go and will restore the changes inside repo.go. :)